src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1432,3 +1460,13 @@ def list_consumption_locations():
     client = web_client_factory()
     regions = client.list_geo_regions(sku='Dynamic')
     return [{'name': x.name.lower().replace(" ", "")} for x in regions]
+
+
+def _set_arr_affinity(name, resource_group_name, config, slot=None):
+    client = web_client_factory()
+    if not slot:
+        updated_config = client.web_apps.create_or_update(resource_group_name, name, config)
+    else:
+        print("Slot update arr Affinity")
```

Comment:
Was this for testing?  It should be removed.

Reply:
yes this was for testing i will remove this.